### ISSUE TITLE
Update mock data (for fallback and testing) from BFF / backend

### ISSUE THREAD

**Issue by seanes** [digdir/dialogporten-frontend#1188]
**Tl;dr**

Update mock data used in Arbeidsflate, for mocking Dialogporten. Current set is outdated and endpoints are missing.

Consider using examples for user 14886498226 in tt02 - with minor modications.

We should also consider enabling mock data using a different flag (env build var, perhaps?) instead of using `mock=true` as query parameter.

**Motivation**

1) For automation of testing we should provide a semi-rich data set decoupled from actual backend services, through `msw`, update with current schema of dialogporten and BFF. 

2) In case Dialogporten in dev is out of service / unavailable
